cts-java/src/test/java/org/ga4gh/cts/api/TestData.java

```diff
@@ -30,7 +31,7 @@ private TestData() {
      * The default ID of the dataset that holds the test data.  We use something readable so the
      * meaning is clear, but in reality the value of this is unlikely to be human-readable.
      */
-    public static final String DEFAULT_DATASET_ID = "compliance-dataset";
+    public static final String DEFAULT_DATASET_ID = "WyJicmNhMSJd";
```

Comment:
Although it's one less configuration to pass, it suggests a level of relationship to the reference server that we probably shouldn't add.